src/main/java/org/tuberlin/de/geodata/Coordinate.java

```diff
@@ -0,0 +1,15 @@
+package org.tuberlin.de.geodata;
+
+/**
+ * Created by gerrit on 04.11.15.
+ */
+public class Coordinate {
+    public double x;
+
+    @Override
+    public String toString() {
+        return x + "," + y;
+    }
+
+    public double y;
```

Comment:
I consider it a good practice to write variable first, then the methods... You write one variable, then a method and finally another variable... It should be variable, variable and then the method...